posthog/tasks/usage_report.py

```diff
@@ -135,6 +136,13 @@ class UsageReportCounters:
     survey_responses_count_in_period: int
     # Data Warehouse
     rows_synced_in_period: int
+
+    # Data Warehouse metadata
+    active_external_data_schemas_in_period: int
+
+    # Batch Exports metadata
+    active_batch_exports_in_period: int
```

Comment:
May want to rename this to not in period as this metric is really "active at the time this is run"